cmd/wnode/main.go

```diff
@@ -197,6 +197,8 @@ func initialize() {
 		if len(*argIP) == 0 {
 			argIP = scanLineA("Please enter your IP and port (e.g. 127.0.0.1:30348): ")
 		}
+	} else if *fileReader {
+		*bootstrapMode = true
```

Comment:
why would I need to force bootstrap mode here? if one server is a file sharing service available to all, its clients might not want to create their own network

Reply:
filereader does not send or receive messages. it only loads encrypted stuff from disc and decrypts. this mode is necessary for diagnostics.